server/test/articles.spec.js

```diff
@@ -85,6 +85,133 @@ describe('Article endpoints', () => {
           done();
         });
     });
+
+    it('Should return offset must be a number', (done) => {
+      chai.request(app)
+        .get('/api/articles?offset=')
+        .set('Authorization', 'Bearer <token>')
```

Comment:
I think this Authorization header is useless when testing an endpoint which does not require authentication. Removing it would be better.

Reply:
Good, I will take care of it. But I still need clarification about the access of this endpoint, cuz even an authenticate user need to access it